### PR TITLE
Reports — For #2972

### PR DESCRIPTION
**Pull request by WildcardSearch**
**[Ready for testing]**

For #2972

**Templates:**
- [x] report
- [x] report_reasons
- [x] report_reason
- [x] report_duplicate
- [x] report_error
- [x] report_error_nomodal
- [x] report_thanks

```diff
+ report -> report.twig
+ report_thanks -> report_thanks.twig
+ report_reasons -> report_reasons.twig
+ report_error_nomodal -> report_error_nomodal.twig
- report_error
- report_duplicate
- report_reason
```

### CODE REVIEW

**Comment by WildcardSearch** [comment]
@mybb/developers this seems to be working correctly. It should be ready to test.

File: report.php [hunk 1]
```diff
@@ -8,279 +8,236 @@
...
  */
 
-define("IN_MYBB", 1);
+define('IN_MYBB', 1);
 define('THIS_SCRIPT', 'report.php');
 
-$templatelist = "report,report_thanks,report_error,report_reasons,report_error_nomodal,forumdisplay_password_wrongpass,forumdisplay_password";
-require_once "./global.php";
-require_once MYBB_ROOT.'inc/functions_modcp.php';
+$templatelist = 'report,report_thanks,report_error,report_reasons,report_error_nomodal,forumdisplay_password_wrongpass,forumdisplay_password';
```
Comment (Matslom) [inline report.php]: This is unnecessary right now, right?

Comment (WildcardSearch) [reply report.php]: Yes. I just got an answer on Discord from Euan. I will handle it tonight.

Comment (WildcardSearch) [reply report.php]: Done @Matslom

**Comment by andrewjs18** [comment]
from the gui side of this, I didn't see any issues.

**Comment by euantorano** [comment]
@WildcardSearch This still has the `s:in-progress` label, but it looks like it's complete to me - is it?

**Comment by WildcardSearch** [comment]
Yes, @euantorano 

I am not up-to-speed on the labels and when they should be applied/removed.

**Comment by euantorano** [comment]
No problem, I'll give it a test 👍

**Comment by WildcardSearch** [comment]
Great!

**Pull request merged**